backend/sozisel/lib/sozisel/model/sessions.ex

```diff
@@ -17,18 +17,24 @@ defmodule Sozisel.Model.Sessions do
   end
 
   @doc """
-  Returns the list of session_templates that belongs to user
+  Returns the list of session_templates that belongs to user and which name matches given pattern.
```

Comment:
What pattern? Isn't that `%whatever%` in the end? Matches a name it more appropriate as this is not a pattern whatsoever.